.readthedocs.yaml

```diff
@@ -2,6 +2,13 @@ version: 2
 
 build:
   os: ubuntu-20.04
+  jobs:
+    post_checkout:
+      - echo `date`
+      # Exit when there aren't changed in docs directory
+      # --quiet exits with a 1 when there **are** changes,
+      # so we invert the logic with a !
+      - "! git diff --quiet origin/main -- docs"
```

Comment:
I like the simplicity of the command :)

The effect would be that documentation is built in all subsequent commits once a commit has changed the `docs/` folder. I think that behavior would have a big effect, saving many many builds moving forwards.

Have you tested it on the `main` branch itself? Comparing a commit in the main branch with the main branch would also make this skip? So that could stop the main branch from building?

I also think that it's worth in the longer run to go in a direction of actually having a `when` syntax with some easy-to-use / "batteries included" kind of options.